src/engine/qcommon/qcommon.h

```diff
@@ -285,7 +288,7 @@ You or the server may be running older versions of the game."
 
 #define PROTOCOL_VERSION       86
 
-#define URI_SCHEME             GAMENAME_STRING "://"
+#define URI_SCHEME             "dgp://"
```

Comment:
Looking at this I am no longer sure that the URI_SCHEME can be engine specific. How should the OS know what game to start when you click a link with an engine specific URI_SCHEME?

Reply:
We can keep `unv://` for the moment with a glorious `FIXME:`, the _protocol_ is by definition something about _others_, so we have to take time to think properly about it, and it must not be a blocker for other things.

Reply:
@DefaultUser as it seems too early to think about protocol uri scheme and I don't want this to block the PR, can you revert this line to `unv://` and add a `FIXME:` comment?

Reply:
After deep thoughts (I'm not sure) and long time passed (for sure), it now feels stupid to me to have the URI_SCHEME to be engine specific. What we need in engine is a way to grok the URI_SCHEME the game developer wants it to grok. It's up to the game developer to properly configure player system to redirect given URI_SCHEME to the engine with proper options (libpath containing the config file telling the URI_SCHEME proto).

So it now looks very OK to me to set that value in the config file.

Maybe name that constant `URI_PROTO` to be more explicit than `GAMENAME_STRING`.
Years later I completely forgotten what `GAMENAME_STRING` meant and I had to read the code to understand it was the URI proto string.